components/brave_rewards/resources/android_page/components/settingsPage.tsx

```diff
@@ -325,6 +332,37 @@ class SettingsPage extends React.Component<Props, State> {
     }
   }
 
+  renderPromos = () => {
+    const { currentCountryCode, ui } = this.props.rewardsData
+    const { promosDismissed } = ui
+
+    return (
+      <>
+        {getActivePromos(this.props.rewardsData, true).map((key: PromoType) => {
+           if (promosDismissed && promosDismissed[key]) {
+             return null
+           }
+
+           const promo = getPromo(key, this.props.rewardsData) as Promo
```

Comment:
Can we avoid the type assertion here (the `as`)? Ideally we can make TS happy without disabling the type checker.

I see that we're spreading this into the component props below. One suggestion would be to not use spread there and set the props individually instead - spreading has the downside of making it difficult to see what props are getting pushed into the component.

Reply:
Good points, fixed.